tower/src/lib.rs

```diff
@@ -219,6 +219,7 @@ pub use tower_layer::Layer;
 pub use tower_service::Service;
 
 #[allow(unreachable_pub)]
+#[cfg(any(feature = "balance", feature = "discover", feature = "full", feature = "make"))]
```

Comment:
No need to check for `full`, since `full` implies the other three features you're checking for here.

Reply:
Noted, thank you for the advice.

Reply:
Removed the `full` feature flag.